README.rst

```diff
@@ -77,6 +77,9 @@ Extras
    * - ``snowflake``
      - ``pip install 'astronomer-providers[snowflake]'``
      - Snowflake
+   * - ``apache.livy``
```

Comment:
Could we move to this after Amazon one so we have alphabetical ordering after `all` extra

Reply:
sure